tests/e2e/support/api/graph/userManagement.ts

```diff
@@ -129,3 +131,27 @@ export const addUserToGroup = async ({
   })
   checkResponseStatus(response, 'Failed while adding an user to the group')
 }
+
+export const assignRole = async (admin: User, id: string, role: string): Promise<void> => {
+  const applicationEntity = await getApplicationEntity(admin)
+
+  if (!roleStore.has(role)) {
+    applicationEntity.appRoles.map((key, value) => {
+      return roleStore.set(key.displayName, key.id)
+    })
```

Comment:
if you are not using the array returned by map
```suggestion
    applicationEntity.appRoles.forEach((role) => {
      roleStore.set(rome.displayName, role.id)
    })
```